freeglut/freeglut/include/GL/freeglut_ext.h

```diff
@@ -175,6 +175,15 @@ FGAPI void    FGAPIENTRY glutSetWindowData(void* data);
 FGAPI void*   FGAPIENTRY glutGetMenuData( void );
 FGAPI void    FGAPIENTRY glutSetMenuData(void* data);
 
+/*
+ * Clipboard access
+ */
+#define GLUT_CLIPBOARD 0
+#define GLUT_PRIMARY 1
+#define GLUT_SECONDARY 2
+FGAPI void         FGAPIENTRY glutSetClipboard(int selection, const char *string);
+FGAPI const char * FGAPIENTRY glutGetClipboard(int selection);
+
```

Comment:
I don't see anything wrong with this API, but I think every new API we add should be discussed in the mailing list first, because we'll be stuck with it for ever. This github mirror is not the standard place to take decisions about public interfaces.

Could you please post in the freeglut-developers mailing list with your proposal for the new APIs? It's better, and more direct, if you do it.